sdk/tests/utilities/test_api_client_builder.py

```diff
@@ -40,15 +39,16 @@ def test_missing_from_config_file_throws(self, _, missing_attributes, token):
         proxy_config = ProxyConfig(**{
             key.replace("proxy_", ""): value for key, value in source_config_details.items() if
             value is not None and "proxy" in key
-        }) if source_config_details["proxy_address"] is not None else None
+        }) if source_config_details.get("proxy_address", None) is not None else None
 
-        api_config_kwargs = {key: value for key, value in source_config_details.items() if
-                             value is not None and "proxy" not in key}
+        api_config_kwargs = {key: "value" for key, value in config_keys.items() if "proxy" not in key}
```

Comment:
Why did you remove if `value is not None`?

Reply:
it changed from `source_config_details.items()` to `config_keys.items()` as the source of the key/values.  There are no values in `config_keys`.

Reply:
Roger.